.github/workflows/continuous-build.yml

```diff
@@ -0,0 +1,44 @@
+name: "Continuous Build"
+
+on:
+  pull_request:
+    branches:
+      - master
+  push:
+    branches:
+      - master
+
+jobs:
+  build:
+    name: Build
+    runs-on: ${{ matrix.os }}
+    strategy:
```

Comment:
You may want to use `fail-fast: false`

Reply:
Yeah I thought of it - I think on instrumentation repo we have pretty different test profile for each environment (e.g. `maxVersionForTest`. And some flakiness :) Here I'm expecting fail-fast to generally help rather than hurt since I guess it's more common that one failure == all failure.

Reply:
I think this is correct. When that's not true, it's usually a flaky test, but we don't tend to have very many of those, so failing fast seems like the right strategy. We can definitely revisit if we start getting more flaky tests.